css/src/admin-global.css

```diff
@@ -817,6 +817,10 @@ body.folded .wpseo-admin-submit-fixed {
   color: #d63638;
 }
 
+.yoast-notice-migrated-header {
+  margin-top: 10px;
+}
```

Comment:
This is because of the newly added title on all other/existing pages?

Reply:
Exactly.